### ISSUE TITLE
validation_data_path

### ISSUE THREAD

**Issue by jhaggle** [kurapan/EAST#50]
How should the training data be split between training and validation?

**Comment by kurapan** [comment]
That depends on what data you have. If you have one set of data, you can split it to a training and a validation set as you wish. There's no correct way to split it.
